source/Grid/Grid.test.js

```diff
@@ -120,43 +120,49 @@ describe('Grid', () => {
   describe('shows and hides scrollbars based on rendered content', () => {
     const scrollbarSize = getScrollbarSize()
 
+    function renderTwice (markup) {
```

Comment:
This is why I previous suggested what I did- initializing it in `componentWillMount` and then...
> * Adding a check in `componentDidMount` for an undefined `this._scrollbarSize` and re-initializing if necessary

This would handle the server-side rendering concern.

We should not require external code (test code or production code) to call `forceUpdate` in order for the correct layout behavior. `Grid` should update itself if it detects that condition.